Public/Src/FrontEnd/UnitTests/Script.Interpretation/InterpreterTests.cs

```diff
@@ -488,18 +490,21 @@ namespace M
             result.ExpectNoError();
             result.ExpectValues(count: 3);
 
+            string separator = ";";
             Func<string, string> unNormalizePath =
                 path =>
                 {
                     Contract.Requires(!string.IsNullOrWhiteSpace(path));
-                    return path.Replace("/", "\\").Replace("\\;", ";").Replace("p`", string.Empty).Replace("`", string.Empty);
+                    return path.Replace("/", "\\").Replace("\\;", separator).Replace("p`", string.Empty).Replace("`", string.Empty);
                 };
 
             Func<string, string> toUpperInvariant = s => string.IsNullOrWhiteSpace(s) ? s : s.ToUpperInvariant();
 
             var userName = Environment.GetEnvironmentVariable("USERNAME");
             var sysRoot = Environment.GetEnvironmentVariable("SystemRoot");
-            var paths = Environment.GetEnvironmentVariable("Path");
+            var paths = Environment.GetEnvironmentVariable("Path");          
+            var pathValues = paths.Split(new[] { separator }, StringSplitOptions.RemoveEmptyEntries);
+            var absolutePaths = pathValues.Aggregate((currentAccumPaths, nextPath) => currentAccumPaths + separator + Path.GetFullPath(nextPath));

```

Comment:
>+ [](start = 104, length = 1)

Fine for unittest, but in product code we should use a string builder since this is sum(1..n) allocations